Manual configuration in README. (#13)
scipp/copier_template: README.md

pip install -e .
git remote add origin git@github.com:scipp/myproject.git
```

## Manual Configuration
Once the project is built from the template, there are manual settings to be configured per project/repository for deployment of documentation and packages.

### Document Deployment

> 1. Select branch of deployed document.
>   The document will be deployed from a branch via github action.
>   The branch of the document should be selected manually on github.

Reviewer: ```suggestion
1. Select branch of deployed documentation.
  The documentation will be deployed from a branch via GitHub action.
  The branch of the documentation must be selected manually on GitHub.
```

  Go to `Settings > Pages` and set `source` as  `Deploy from a branch` and `Branch` as `gh-pages`.

2. Add `.nojekyll` file.
  Once the document is deployed to a branch, make sure it has `.nojekyll` file in the `gh-pages` branch and you should be able to see the documentation at `scipp.github.io/project_name`.

3. Deployment key.

>   The deployment key should be already set by organization if it is under `scipp` organization.

Author: Sorry I made it confusing, an `ssh key` was needed to deploy it to the `branch`. 
And, you're right, the deployment of documentation from branch is done by github.

### Package Deployment
See (releasing scipp)[https://scipp.github.io/reference/developer/releasing-scipp.html#updating-an-expired-anaconda-token] for more information about deployment.

1. Conda
Go to Settings > Secrets > Actions > Organization secrets.
There is `scipp` organization-wide anaconda key, `ANACONDATOKEN`. But it should be enabled per repository.

2. Pypi
Go to Settings > Secrets > Actions > Repository secrets.
`PYPI_TOKEN` should be configured per repository.
You can make one under your account on (`pypi.org`)[https://pypi.org/].

> If it is the first time of deployment, the token needs to have all-access and after the first deployment, it can be replaced with the new token that has the access to the project.

Reviewer: This was very confusing for me. I think I had to create a personal token? And the first upload was done manually with my username and password. Is there a better way? In any case, may be worth describing in more detail.
